pyproject.toml

```diff
@@ -35,6 +38,9 @@ pysqlar = "^0.1.2"
 memory-profiler = "^0.60.0"
 matplotlib = "^3.6.0"
 
+[tool.poetry.group.dev.dependencies]
+jupyter = "^1.0.0"
```

Comment:
do we still need Jupyter here?

Reply:
I moved the dev dependencies to the dev group, because the old method was deprecated